### ISSUE TITLE
Google now broken?

### ISSUE THREAD

**Issue by mypalchris** [simonray/exchange-rate#2]
Hey - It looks like there is something up with the Google API - I'm just getting a server error. Have Google done the same as Yahoo and pulled the API?

**Comment by ghost** [comment]
Yes I'm facing the same issue. Did you find any work around? @mypalchris @simonray
